### PR TITLE
Add python to path

### PR DESCRIPTION
**Pull request by scaryrawr**
- Add custom switch to pass in [PrependPath switch](https://docs.python.org/3/using/windows.html#installing-without-ui)
> Adds install and Scripts directories to PATH and `.py` to PATHEXT

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/665)

### CODE REVIEW

**Review by Reset816 — APPROVED** [review]

File: manifests/Python/Python/3.8.3.yaml [hunk 1]
```diff
@@ -15,3 +15,4 @@ Installers:
       Switches:
         Silent: /Quiet
         SilentWithProgress: /Passive
+        Custom: PrependPath=1
```
Comment (Reset12138) [inline manifests/Python/Python/3.8.3.yaml:18]:
```suggestion
        Custom: PrependPath = 1
```
will it looked better?

**Review by Reset816 — CHANGES_REQUESTED** [review]

File: manifests/Python/Python/3.8.3.yaml (hunk 1 above)
Comment (scaryrawr) [reply manifests/Python/Python/3.8.3.yaml:18]: If you add spaces, python does not get added to the path.

Comment (Reset12138) [reply manifests/Python/Python/3.8.3.yaml:18]: got it

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**